### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by addod19**
code review request

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
Hi, @addod19 
<h2>Good first attempt. Needs a bit more of work</h2>
Hello! 👋🏼 You are on the right path, the project looks good. well done working alone. 
Just few things to do;
<li>update the README file</li>
example below; 

image

<li>worked on the nav bar text input and search button</li>
example below; 

image

<li>run your code on (https://validator.w3.org) and fix the errors.</li><br>

Submit a new code review request. Thanks!

Tips:
For every HTML/CSS projects please check your files with the W3 Validator and the CSS Linter

File: assets/style.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    height: 56px;
+    background: white;
+    position: fixed;
+    top: 0px;
+    box-shadow: 1px 1px 1px 1px rgb(240, 240, 240);
+    z-index: 1;
+}
+
+nav > a .apps {
+    position: absolute;
```
Comment (bellom) [inline assets/style.css:39]: Please try your best to have DRY(don't repeat yourself) code.
line 20, 28, 36, 42, 49, 56, 62, 69 could be written in one liner.
example; 
```
nav, a, span {
  position: absolute;
}
```

**Review by bellom — CHANGES_REQUESTED** [review]

**Review by bellom — CHANGES_REQUESTED** [review]
Hi, @addod19 
**_Great work, i can see you now have DRY code, Awesome!! :smile:_**
**_For some reason you didn't work completely on the previous PR corrections._**
Kindly work on the following:
<li>worked on the nav bar text input and search button, 
the search button should be at the end (rightmost part) of the search bar.</li>
example below; 

image

<li>run your code on (https://validator.w3.org) and fix the errors.</li><br>
[i ran your code on w3 validator, you have a lot of errors you need to fix] 

**Free free to message me on slack if you need help.**

Submit a new code review request. Thanks!

Tips:
For every HTML/CSS projects please check your files with the W3 Validator and the CSS Linter

**Review by bellom — APPROVED** [review]
Hi @addod19 
<h2>Great work fixing all the validator errors :+1: </h2>
<h4>Congratulations</h4> you can go ahead and merge to master.

**Comment by addod19** [comment]
project one: YouTube Clone Completed

**Pull request merged**